modules/gear/loadouts/sf_sl.sqf

```diff
@@ -0,0 +1,29 @@
+
+/* Common Special Forces Gear */
+#include "sf_common.sqf"
+
+/* Item Slots */
+ADD_WEAPON("ACRE_PRC148");
+ADD_WEAPON("ACE_HuntIR_monitor");
+
+/* Primary Magazine Slots */
+ADD_MAGAZINE("30Rnd_556x45_Stanag", 5);
+/*ADD_MAGAZINE("30Rnd_556x45_StanagSD", 7);*/
+ADD_MAGAZINE("ACE_30Rnd_556x45_T_Stanag", 2);
+ADD_MAGAZINE("1Rnd_HE_M203", 4);
+ADD_MAGAZINE("ACE_HuntIR_M203", 1);
```

Comment:
I'm pretty sure this will boot either medical supplies or a smoke grenade out of their inventory. Add the HuntIR round to their ruck or something else to offset it

Reply:
Okay I'll change that.